azure/services/virtualmachines/virtualmachines.go

```diff
@@ -47,6 +49,8 @@ type VMScope interface {
 	SetProviderID(string)
 	SetAddresses([]corev1.NodeAddress)
 	SetVMState(infrav1.ProvisioningState)
+	SetConditionTrue(clusterv1.ConditionType)
```

Comment:
That makes sense! It was leftover from when I tried to set a `VMUnhealthy` condition to True instead of setting `VMRunning` to false. I'll go ahead and remove it!